src/renderer/components/dock/dock-store/dock.store.ts

```diff
@@ -97,6 +97,11 @@ export interface DockTabCloseEvent {
   tabId: TabId; // closed tab id
 }
 
+export interface DockTabsNumberEvent {
```

Comment:
Some leftovers from removed reaction event? Or is  it used somewhere? 🧐

Reply:
Good catch